### PR TITLE
FIX: import multisig from some formats

### PR DESCRIPTION
**Pull request by Overtorment**

### CODE REVIEW

File: blue_modules/fs.js [hunk 1]
```diff
@@ -87,12 +87,15 @@ const showFilePickerAndReadFile = async function () {
           : [DocumentPicker.types.allFiles],
     });
 
+    const uri = Platform.OS === 'ios' ? decodeURI(res.uri) : res.uri;
```
Comment (Overtorment) [inline blue_modules/fs.js:90]: fyi @marcosrdz

Comment (Overtorment) [reply blue_modules/fs.js:90]: is that ok..?

Comment (marcosrdz) [reply blue_modules/fs.js:90]: Afaik you know use decodeURI for both platforms

Comment (Overtorment) [reply blue_modules/fs.js:90]: for some reason on my android emu  it didnt work for a file with space in it  o_O

**Review by marcosrdz — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/bp0g35ha4b4j9avq817k8m8xpg

[download apk](http://filestorage.bluewallet.io:1488/1604505206.apk)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**